docs/src/man/hosting.md

```diff
@@ -171,21 +171,21 @@ workflow configuration file:
 ```yaml
 name: Documentation
 
-on: [push]
+on:
+  push:
+    branches:
+      - master
+    tags: '*'
+  pull_request:
```

Comment:
```suggestion
  pull_request: # only needed for preview builds (push_preview=true in deploydocs)
```

Reply:
Isn't this also required for running documenter in PRs from forks? (My post in the OP was a bit misleading.)

I guess people familiar with Travis would find this setup natural.

Reply:
Ah, right.